alt_exit.go

```diff
@@ -51,6 +51,16 @@ func Exit(code int) {
 	os.Exit(code)
 }
 
+// handleExit runs all the Logrus atexit handlers and then terminates the program using the logger's Exit function (or os.Exit() if no exit function is set).
+func (logger *Logger) handleExit(code int) {
+	runHandlers()
+	if logger.Exit != nil {
+		logger.Exit(code)
```

Comment:
I agree that this allows a library user to change `Fatal`'s behavior to instead not exit. I don't see that as a bad thing. But if the consensus is that it *is* a bad thing, we should scrap this PR and instead write one to replace a `Fatal`'s exit with a `panic` (probably via a flag).